add-remove.test.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import TodoList from './src/modules/addTodo.js';
+import LocalStorage from './mocks/LocalStorage.js';
+
+document.body.innerHtml = `<main>
+<div id="list-container">
+        <div class="list-head">
+            <p class="todo">Today's To Do List</p>
+
+           <a class="reload-btn reload-icon"> <i id="icon " class="fa-solid fa-rotate fa-lg font-awesome-icon"></i></a>
+        </div>
+       <div class ="input-container">
+         <input type="text" class="input" name="addtext" placeholder="Add to your list..." />
+        <i id="add-icon" class="fa fa-plus-square fa-lg font-awesome-icon"></i>
+        </div>
+        <div id="list-body"></div>
+        <div class="button" id="clear">
+            <button>Clear all completed</button>
+        </div>
+    </div>`;
+
+global.localStorage = new LocalStorage();
```

Comment:
- [ ] There is an error raised while running the jest testing library, So please fix that.
see the image below for more info.

![image](https://user-images.githubusercontent.com/68796232/178766534-d5c3f048-df26-4439-ae67-988b3db05997.png)

Reply:
@Omar-Muhamad  I'm not getting this error